src/briefcase/commands/base.py

```diff
@@ -143,6 +146,81 @@ def __init__(self, base_path, home_path=Path.home(), apps=None, input_enabled=Tr
         self.logger = Log()
         self.save_log = False
 
+    def check_obsolete_data_dir(self):
+        """Inform user if obsolete data directory exists.
+
+        TODO: Remove this check after 1 JAN 2023 since most users will have transitioned by then
+
+        Previous versions used the ~/.briefcase directory to store
+        downloads, tools, etc. This check lets users know the old
+        directory still exists and their options to migrate or clean up.
+        """
+        dot_briefcase_path = self.home_path / ".briefcase"
+
+        if not dot_briefcase_path.exists():
+            return
+
+        if self.data_path.exists():
+            self.logger.warning(
+                f"""\
+Briefcase is no longer using the data directory:
+
+    {dot_briefcase_path}
+
+This directory can be safely deleted.
+"""
+            )
+        else:
+            self.logger.warning(
+                f"""\
+*************************************************************************
+** NOTICE: Briefcase is changing it's data directory                   **
+*************************************************************************
+
+    Briefcase is moving it's data directory from:
+
+        {dot_briefcase_path}
+
+    to:
+
+        {self.data_path}
+
+    If you continue, Briefcase will re-download the tools and data it
+    uses to build and package applications.
+
+    To avoid potentially large downloads and long installations, you
+    can manually move the old data directory to the new location.
+
+    If you continue and allow Briefcase to re-download it's tools, the
+    old data directory can be safely deleted.
```

Comment:
minor typos (3 occurrences): it's -> its

Reply:
/me grumbles... :-)